components/navbar.tsx

```diff
@@ -163,10 +159,10 @@ const Navbar = () => {
             Contact
           </NavLink>
           <NavLink
-            href="#"
+            href={applicationLink}
             variant={"mobile_bold"}
-            className="bg-[rgb(75,0,0)] cursor-not-allowed" // TODO: Update this when we get applying working
-            // onClick={handleMobileLinkClick}
+            className="bg-[rgb(75,0,0)]"
+            onClick={handleMobileLinkClick}
```

Comment:
Let's make this open in a new tab just like the desktop apply button.
```suggestion
            onClick={handleMobileLinkClick}
            target="_blank"
```